db/redis.js

```diff
@@ -39,8 +47,8 @@ async function clearKeys(){
     await client.flushall()
 }
 
-async function quit(){
-    await client.quit()
+async function disconnect(){
```

Comment:
This was motivating by me getting a failing test when using `quit()` [and also my pendantic-ness of wanting to use `connect` which makes its ending function seem more natural to be `disconnect` rather than `quit`].